app/src/main/java/org/liberty/android/fantastischmemo/ui/QACardActivity.java

```diff
@@ -132,6 +136,54 @@ public MultipleLoaderManager getMultipleLoaderManager() {
         return multipleLoaderManager;
     }
 
+    /* favourites deck */
+    private String favouritesDbPath = "/sdcard/favourites.db";
+    private AnyMemoDBOpenHelper favouritesDbHelper;
+
+
+    public AnyMemoDBOpenHelper getFavouritesDbHelper(){
+        return favouritesDbHelper;
+    }
+
+    public void favouriteCard(Card card){
+        if(favouritesDbHelper == null) {
+            favouritesDbHelper = AnyMemoDBOpenHelperManager.getHelper(getApplicationContext(), favouritesDbPath);
+        }
+
+        favouritesDbHelper.getCardDao().createCard(card);
+    }
+
+    public void unfavouriteCard(Card card){
+        if(favouritesDbHelper == null) {
+            favouritesDbHelper = AnyMemoDBOpenHelperManager.getHelper(getApplicationContext(), favouritesDbPath);
+        }
+
+        favouritesDbHelper.getCardDao().delete(card);
+    }
+
+    public List<Card> getAllFavourites(){
+        if(favouritesDbHelper == null) {
+            favouritesDbHelper = AnyMemoDBOpenHelperManager.getHelper(getApplicationContext(), favouritesDbPath);
+        }
+
+        return favouritesDbHelper.getCardDao().queryForAll();
+    }
+
+    public void emptyFavourtiesDeck(){
```

Comment:
fix typo

Reply:
We will fix it after the merge because we still have to merge Jonathans branch to this merge.